### DISCUSSION TITLE
Resetting permissions for a site

### DISCUSSION THREAD

**Discussion by bbbhltz** [qutebrowser/qutebrowser#7067] [Q&A] [answered]
Hello,

I feel like I have done this before, but imagine you decided not to accept cookies for a site, and you said NEVER to geolocation. How do you reset all of that? I know that on ZOOM I once set it to NEVER allow access to mic, but then reset with no issues whatsover.

The situation is a site I use for work (cannot share the link) and because of bad design, if it does not know your location it blocks you.

I can access it using `--temp-dir` flag.

[version info](https://paste.the-compiler.org/view/7753d4cc)

**Comment by mschilli87** [comment, marked as answer]
Try `:set --pattern=<url> content.geolocation` (with `<url>` being the link in question).

It should tell you

> `content.geolocation = false for <url>`

You can reset it to prompting you via  `:set --pattern=<url> content.geolocation ask` or always allow it for this site via `:set --pattern=<url> content.geolocation true`.

---
*edit*: If you are not sure about the exact URL, have a look at `/home/bbbhltz/.config/qutebrowser/autoconfig.yml` and search for `content.geolocation`. You should have a list with all active patterns just below that, including `<url>: false`.

**Reply by bbbhltz** [reply]
OK, that part is perfect. It now asks for my location again. I 

Now, it still doesn't ask for cookies either... Apparently that is another thing blocking me.

I will set this as solved though, because it is the answer to my main question.

**Reply by mschilli87** [reply]
Like I said: `/home/bbbhltz/.config/qutebrowser/autoconfig.yml` should contain all the pattern-specific settings you have set interactively. Searching that file for the URL in question help you to identify all settings you might want to modify. Alternatively, you could do the same within qutebrowser itself via [qute://configdiff/](qute://configdiff/).
